wolfSSL/src/chapter02.md

```diff
@@ -394,6 +394,134 @@ Removes code for OAEP padding.
 
 Turns off AES-CBC algorithm support.
 
+#### WOLFCRYPT_ONLY
+
+Enables wolfCrypt only while disabling TLS.
+
+#### NO_CAMELLIA_CBC
+
+Disables Camellia CBC support but only applies to TLS cipher suites only.
+
+#### NO_AES
+
+Disables AES algorithm support.
+
+#### NO_AES_128
+
+Used for AES key size selection at compile time.
+
+#### NO_AES_192
+
+Used for AES key size selection at compile time.
+
+#### NO_AES_256
+
+Used for AES key size selection at compile time.
+
+#### NO_AESGCM_AEAD
+
+Used for disabliing TLS cipher suites thst use AES GCM. It is used internally when no AES GCM cipher suites are enabled, but can also be used to limit cipher suites.
+
+#### NO_ASN_TIME
+
+Disables time checking for ASN. Note: This should be used with caution because all certificate begin/end date checking will be skipped.
+
+#### NO_CHECK_PRIVATE_KEY
+
+This macro disables additional private key checking that is on by default. This enables checking to validate the private key is a pair for the public key. It is supported for RSA, ECDSA, ED25519, ED448, Falcon, Dilithium and Sphincs.
+
+#### NO_DH
+
+Disables Diffie-Hellman (DH) support.
+
+#### NO_ED25519_CLIENT_AUTH
+
+Disables TLS client authentication support for ED25519. It is used to reduce memory usage during TLS if ED25519 is not used, since it requires caching messages.
+
+#### NO_ED448_CLIENT_AUTH
+
+Disables client authentification for ED448.
+
+#### NO_FORCE_SCR_SAME_SUITE
+
+By default secure renegotiation requires using the same cipher suite. This disables that requirement.
+
+#### NO_MULTIBYTE_PRINT
+
+Used for compiling out special characters that embedded devices may have problems with.
+
+#### NO_OLD_SSL_NAMES
+
+This disables some of the old OpenSSL compatibility macros for using wolfSSL and OpenSSL together.
+
+#### NO_OLD_WC_NAMES
+
+Removes unneeded namespace.
+
+#### NO_OLD_POLY1305
+
+For compiling out special characters that embedded devices may have problems with.
```

Comment:
This disables support for the old ChaCha20/Poly1305 TLS v1.2 cipher suite that is typically used for interop. This is the cipher suite added prior to the official adoption. @ejohnstown can shed additional light on this.